presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkRow.java

```diff
@@ -17,18 +17,44 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class SerializedPrestoSparkPage
+public class PrestoSparkRow
         implements Serializable
 {
+    private static final int INSTANCE_SIZE = Long.BYTES * 2 /* headers */
```

Comment:
Why not using `ClassLayout.parseClass` as we did for other Presto class? -- I am asking this is just adding these bytes together doesn't necessarily sum up to the instance size ,  e.g. there are Java object padding size, etc.

Reply:
The Jol library is not in the Spark's classpath, and I don't want to add one to avoid potential clashes.

Reply:
@arhimondr : Sounds reasonable, just add a comment explain why this is the case. 

Also, looks like JOL and your calculation returns different results? (You seems to over-estimate): https://gist.github.com/wenleix/faae51ac5fe3c0f688d8520c7c0d3105

```
Instance Size from JOL: 24
Instance Size from Andrii: 32
```